cli/cli.go

```diff
@@ -2,43 +2,105 @@ package cli
 
 import (
 	"fmt"
-	"github.com/shpota/goxygen/codegen"
 	"io"
 	"regexp"
+	"strings"
+
+	"github.com/shpota/goxygen/codegen"
 )
 
 func Start(w io.Writer, commands []string) {
-	if len(commands) == 1 && commands[0] == "help" {
+	argsLen := len(commands)
+	if argsLen == 1 && commands[0] == "help" {
 		fmt.Fprintln(w, usage)
 		return
 	}
-	if len(commands) == 2 && commands[0] == "init" {
-		projectName := commands[1]
-		validName, _ := regexp.MatchString("^[a-zA-Z0-9_-]+$", projectName)
-		if validName {
-			codegen.Generate(projectName)
+	if (argsLen == 2 || argsLen == 3) && commands[0] == "init" {
+		var frontendTech string
```

Comment:
You could set it straight away to `React` so that you don't have to do it later.

Reply:
i think it's ok both way. But this is clear